lib/ansible/modules/cloud/amazon/batch_compute_environment.py

```diff
@@ -0,0 +1,496 @@
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: batch_compute_environment
+short_description: Manage AWS Batch Compute Environments
+description:
+    - This module allows the management of AWS Batch Compute Environments.
+      It is idempotent and supports "Check" mode.  Use module M(batch_compute_environment) to manage the compute
+      environment, M(batch_job_queue) to manage job queues, M(batch_job_definition) to manage job definitions.
+
+version_added: "2.4"
+
+author: Jon Meran (@jonmer85)
+options:
+  compute_environment_name:
+    description:
+      - The name for your compute environment. Up to 128 letters (uppercase and lowercase), numbers, and underscores
+        are allowed.
+    required: true
+
+  type:
+    description:
+      - The type of the compute environment.
+    required: true
+    choices: ["MANAGED", "UNMANAGED"]
+
+  state:
+    description:
+      - Describes the desired state.
+    required: true
+    default: "present"
+    choices: ["present", "absent"]
+
+  compute_environment_state:
+    description:
+      - The state of the compute environment. If the state is ENABLED, then the compute environment accepts jobs
+        from a queue and can scale out automatically based on queues.
+    default: "ENABLED"
+    choices: ["ENABLED", "DISABLED"]
+
+  service_role:
+    description:
+      - The full Amazon Resource Name (ARN) of the IAM role that allows AWS Batch to make calls to other AWS
+        services on your behalf.
+    required: true
+
+  compute_resource_type:
+    description:
+      - The type of compute resource.
+    required: true
+    choices: ["EC2", "SPOT"]
+
+  minv_cpus:
+    description:
+      - The minimum number of EC2 vCPUs that an environment should maintain.
+    required: true
+
+  maxv_cpus:
+    description:
+      - The maximum number of EC2 vCPUs that an environment can reach.
+    required: true
+
+  desiredv_cpus:
+    description:
+      - The desired number of EC2 vCPUS in the compute environment.
+
+  instance_types:
+    description:
+      - The instance types that may be launched.
+    required: true
+
+  image_id:
+    description:
+      - The Amazon Machine Image (AMI) ID used for instances launched in the compute environment.
+
+  subnets:
+    description:
+      - The VPC subnets into which the compute resources are launched.
+    required: true
+
+  security_group_ids:
+    description:
+      - The EC2 security groups that are associated with instances launched in the compute environment.
+    required: true
+
+  ec2_key_pair:
+    description:
+      - The EC2 key pair that is used for instances launched in the compute environment.
+
+  instance_role:
+    description:
+      - The Amazon ECS instance role applied to Amazon EC2 instances in a compute environment.
+    required: true
+
+  tags:
+    description:
+      - Key-value pair tags to be applied to resources that are launched in the compute environment.
```

Comment:
Unfortunately that is correct. I believe you need to create a new Compute Environment to modify tags